app/src/main/res/xml/network_security_config.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<network-security-config>
+    <base-config cleartextTrafficPermitted="true">
+        <trust-anchors>

```

Comment:
You should check the "Insecure Base Configuration" warning and see what it means to try to fix it.

Reply:
I managed to remove it actually, I needed this code because the connection to openWeather for the icon is HTTP by default (not secure) so we have to manually allow the system to connect to unsecure URLs. Now the icon is fetched via a HTTPS URL which is secure and we don't need to change any settings, so this file becomes redundant